### ISSUE TITLE
Add a config option to deny mob spawns: Forge options only pushes players away!

### ISSUE THREAD

**Issue by ShelLuser** [Mithion/Mana-And-Artifice#1018]
TL;DR => adding a Forge biome modifier is _a lot_ of hassle and I think you never stopped to think about the burden you're putting on us players. It also doesn't help that the Forge documentation isn't very consistent and on top of **that**: you'll even discover Minecraft limitations.

**No** offense intended: while I was working out all of the above ...  I just changed a config option for "another mod" to make it work as I wanted it to. But here...  honestly: the only reason I am still here is because my gf & me believe that your mod is one of the best magic mods out there. But your implementation?  It's kinda bad, really.

**Description**

In my opinion your gameplay balancing is pretty flawed: hulker zombies have no place in the Nether if you ask me. Of course that is _my_ opinion, but it does lead up to: how do I remove this mess? (being dead honest: sorry, no sugar coating).

The problem here is that Forge biome modifiers are hardly as easy as you may think:

https://forge.gemwire.uk/wiki/Biome_Modifiers

Doable?  For sure. Not gonna lie here: I learned a lot of good stuff pursuing this goal. Just keep in mind: the _only_ reason I went this far is because Aya & me _love_ this mod. Not everyone does? Not gonna lie: the first time we stepped into the nether and noticed the what we believe to be an unbalanced mess: we ran. When I spoke up on Discord.. everyone was indifferent. Worse yet: no one really knew how to fix this, removing mod spawns?

You **do** realize that most players don't want to bother and will just replace this for someting else?

If you ask us: this is one of the best magic mods there is. EXCEPT.. for the mess in the Nether & End. And I don't think it does you any good that us admins have to dig through a ton of messy documentation to get things to work. Not gonna lie: it can also makes us better admins.

.... but I _also_ doubt it's gonna do the mod much good for acceptance either.

This suggestion isn't really for us...  

![datapack](https://github.com/user-attachments/assets/cc78cb9e-2bfa-4869-8a19-90f41110ea81)

^ I made it work, and this is awesomesauce.

But how many players are going to see the facts (IMO: unbalanced mess: same mobs from Overworld in the Nether) and just give up?  You'll say "use a datapack".. most mods _don't_ require me to study the actual Forge mechanics to make things work ya know.

Why isn't there a tag for nether spawn denial?  Or end denial?  With the _utmost_ respect: the lack of those options almost makes me wonder if you really know how players can cope with all this; it's always easy when looking at things in hindsight or.. as a developer who knows all the mechanics inside out.

Just our 2 cents?

**Comment by ShelLuser** [comment]
So we (I play closely with my gf).. have given this many thoughts and I even discussed this suggestion with her. We came to the conclusion that my suggestion is probably kinda obsolete. 

At the risk of coming across as arrogant:
- If you set up a modpack.. isn't this kind of a must-know procedure? *
- Why ignore existing standards instead of empowering them?

* At the rist of coming accross as arrogant...  but I've been there, I even ignored this mod for months because it was "obviously" unbalanced. Then I learned about Forge's biome modifiers. The rest is history.

So I'm closing this as a non-issue (in hindsight).
